README.rst

```diff
@@ -1 +0,0 @@
-Browse the branches to see the different notebooks I've been working on!
```

Comment:
It'd be great to have some more info in the readme – perhaps a link to your blog, plus a link to the projects page to point out your goals/progress?